index.html

```diff
@@ -8,12 +8,92 @@
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
+ </head>
 <body>
 
+
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
-
+    <header class="header">
+        
+          <img src="img/karma-logo.svg" alt="image-logo" class="logo"/>
+        
+        <nav class="nav_bar">
+            <ul class="navigation_list">
+                <li class="navigation_item">
+                    <a  id="meet_karma" class="navigation_link" href="#"><strong>Meet Karma</strong></a> 
+                </li>
+                <li class="navigation_item">
+                    <a class="navigation_link" href="#">How it Works</a> 
+                </li>
+                <li class="navigation_item">
+                    <a class="navigation_link" href="#">Store</a> 
+                </li>
+                <li class="navigation_item">
+                    <a class="navigation_link" href="#">Blog</a> 
+                </li>
+                <li class="navigation_item">
+                    <a class="navigation_link" href="#">Help</a> 
+                </li>
+                <li class="navigation_item">
+                    <a class="navigation_link" href="#">Login</a> 
+                </li>
+            </ul>
+        </nav>
+        <!-- <div class="hamburger">
+            <a href="#" onclick="toggleMenu()">
+                <img src="./img/menu-hamburger.svg" alt="toolbar"/>
+            </a>
+        </div> -->
+    </header>
+          <!--MAIN-->
+  <main class="main">
+         <!-- Intro Intro Karma Container -->
+         <div class="intro_karma">
+              <h1>Introducing Karma</h1>
+              <h2>Bring WiFi with you,everywhere you go. </h2>
+              <button>Learn More</button>
+         </div>
+          <!--- Need Karma Container-->
+         <div class="need_karma">
+            <h3>Everyone needs a little Karma. </h3>
+            <div class="features">
+                <div class="feature_icon">
+                   <img  src="img/icon-devices.svg" alt="device" class="icon"/>
+                   <p>Internet for all devices</p>
+                </div>
+                <div class="feature_icon">
+                    <img src="img/icon-coffee.svg" alt="coffee" class="icon"/>
+                   <p>Boost your productivity </p>
+                </div>
+                <div class="feature_icon">
+                    <img src="img/icon-refill.svg" alt="refill" class="icon"/>
+                    <p>Pay as You Go</p>
+                </div> 
+            </div>
+         </div>
+            <hr>
+               <!---------Footer Container-------->
+              <footer class="footer">
```

Comment:
The footer is not part of the main container, since it does not contain any useful contents. Therefore, I would put this after the closing of `<main>` tag